bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/model/Temperature.java

```diff
@@ -63,7 +63,8 @@
 	@ProviderMappings({ 
 			@Provider(name = ProviderName.WUNDERGROUND, property = "current_observation.dewpoint_c"),
 			@Provider(name = ProviderName.FORECASTIO, property = "dewPoint"),
-			@Provider(name = ProviderName.HAMWEATHER, property = "dewpointC")
+			@Provider(name = ProviderName.HAMWEATHER, property = "dewpointC"),
+			@Provider(name = ProviderName.WORLDWEATHERONLINE, property = "DewPointC")
```

Comment:
No - the property is used to literally map to a JSON key from the weather providers response:
... 
                       "cloudcover": "97",
                        "DewPointC": "5",
                        "DewPointF": "41",
                        "FeelsLikeC": "15",
 ...
So for WorldWeatherOnline it's ok. :-)